src/lqcut.jl

```diff
@@ -98,13 +90,13 @@ function lq_drift_time_correction(
     lq_cut = lq_DEP[lq_lower .< lq_DEP .< lq_upper .&& t_lower .< dt_eff_DEP .< t_upper]
     t_cut = dt_eff_DEP[lq_lower .< lq_DEP .< lq_upper .&& t_lower .< dt_eff_DEP .< t_upper]
 
-    #linear fit
-    result_µ, report_µ = chi2fit(1, t_cut, lq_cut; uncertainty=true)
-    parameters = mvalue(result_µ.par)
-    drift_time_func(x) = parameters[1] .+ parameters[2] .* x
+    #polynomial fit
+    result_µ, report_µ = chi2fit(pol_fit_order, t_cut, lq_cut; uncertainty=true)
+    par = mvalue(result_µ.par)
+    drift_time_func(x) =  sum((mvalue(par[i])) * x^(i-1) for i in eachindex(par))
```

Comment:
Is this supposed to just be the Polynomial? Because for that there is the `PolCalFunc` which you can use. In addition, I think the `chi2` report contains the function too.